### ISSUE TITLE
[Bug] `NullPointerException: No chunk exists at [0, 0]`

### ISSUE THREAD

**Issue by SettingDust** [SuperMartijn642/Formations#3]
**Version Info**
- Minecraft, 1.20.1
- Formations, 1.0.1

**What mod loader are you using?**: Fabric

**Description of the Bug**
```
[11:37:21] [Worker-Main-16/ERROR] (formations) Encountered an exception whilst processing block '{state=Block{minecraft:grass_block}[snowy=false], nbt=null}' with processor of class 'class com.supermartijn642.formations.structure.processors.BiomeReplacementProcessor'!
java.lang.NullPointerException: No chunk exists at [0, 0]
	at net.minecraft.world.ChunkRegion.getBlockState(ChunkRegion.java:3088) ~[client-intermediary.jar:?]
	at com.supermartijn642.formations.structure.processors.BiomeReplacementProcessor.processBlock(BiomeReplacementProcessor.java:569) ~[formations-1.0.1-fabric-mc1.20.jar:?]
	at net.minecraft.structure.StructureTemplate.localvar$ckc000$formations$processBlockInfos(StructureTemplate.java:2334) ~[client-intermediary.jar:?]
	at net.minecraft.structure.StructureTemplate.process(StructureTemplate.java) ~[client-intermediary.jar:?]
	at net.minecraft.structure.StructureTemplate.place(StructureTemplate.java:225) ~[client-intermediary.jar:?]
	at net.minecraft.structure.pool.SinglePoolElement.generate(SinglePoolElement.java:124) ~[client-intermediary.jar:?]
	at net.minecraft.structure.PoolStructurePiece.generate(PoolStructurePiece.java:89) ~[client-intermediary.jar:?]
	at net.minecraft.structure.PoolStructurePiece.generate(PoolStructurePiece.java:85) ~[client-intermediary.jar:?]
	at net.minecraft.structure.StructureStart.place(StructureStart.java:100) ~[client-intermediary.jar:?]
	at net.minecraft.world.gen.chunk.ChunkGenerator.method_38265(ChunkGenerator.java:320) ~[client-intermediary.jar:?]
	at com.google.common.collect.ImmutableList.forEach(ImmutableList.java:422) ~[guava-31.1-jre.jar:?]
	at net.minecraft.world.gen.chunk.ChunkGenerator.generateFeatures(ChunkGenerator.java:319) ~[client-intermediary.jar:?]
	at net.minecraft.world.chunk.ChunkStatus.method_51375(ChunkStatus.java:108) ~[client-intermediary.jar:?]
	at net.minecraft.world.chunk.ChunkStatus$SimpleGenerationTask.doWork(ChunkStatus.java:309) ~[client-intermediary.jar:?]
	at net.minecraft.world.chunk.ChunkStatus.runGenerationTask(ChunkStatus.java:252) ~[client-intermediary.jar:?]
	at net.minecraft.server.world.ThreadedAnvilChunkStorage.method_17225(ThreadedAnvilChunkStorage.java:684) ~[client-intermediary.jar:?]
	at com.mojang.datafixers.util.Either$Left.map(Either.java:38) ~[datafixerupper-6.0.8.jar:?]
	at net.minecraft.server.world.ThreadedAnvilChunkStorage.method_17224(ThreadedAnvilChunkStorage.java:673) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at net.minecraft.server.world.ChunkTaskPrioritySystem.method_17634(ChunkTaskPrioritySystem.java:62) ~[client-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.runNext(TaskExecutor.java:91) ~[client-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.runWhile(TaskExecutor.java:146) ~[client-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.run(TaskExecutor.java:102) ~[client-intermediary.jar:?]
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
```

**Steps to Reproduce**
Don't know

**Comment by SuperMartijn642** [comment]
What mod loader are you using?

**Comment by SettingDust** [comment]
> What mod loader are you using?

Updated. Fabric

**Comment by SuperMartijn642** [comment]
I fixed it now in Formations version 1.0.2.
Thank you for reporting the issue!
